### ISSUE TITLE
Test for challenge-1-decentralized-staking does not exist

### ISSUE THREAD

**Issue by pontus-dev** [scaffold-eth/scaffold-eth-typescript-challenges#25]
In the vanilla JavaScript version of this challenge there is a file `scaffold-eth-challenges/packages/hardhat/test/challenge_1.js`. This file is missing in this TypeScript version of the challenge. Instead, there is only a sample test file `scaffold-eth-typescript-challenges/packages/hardhat-ts/test/sample_test.ts` which does not work due to `chai` not being installed as a package. It also does not test the challenge code.

In other words: 
We need a version of `challenge_1.js` in this TypeScript version of the challenge. Preferably made into a TypeScript version.

**Comment by pontus-dev** [comment]
Added this pull request to fix the issue:
https://github.com/scaffold-eth/scaffold-eth-typescript-challenges/pull/26
